### ISSUE TITLE
Additional `<lb/>` at the end of (ostensibly) verse lines?

### ISSUE THREAD

**Issue by joeytakeda** [internetshakespeare/ise2-import#4]
I'm not sure what this is saying (from iseMV_F1):

```
 <lb type="tln" n="27"/>Would blow me to an Ague, when I thought<lb/>
 <lb type="tln" n="28"/>What harme a winde too great might doe at <g ref="g:longS">ſ</g>ea<lb/>
```

Why do we need to the two `<lb/>`s? What are the non-typed`<lb/>`s? Do they signal the line beginning for this edition?

**Comment by martindholmes** [comment]
I've always assumed the tln is canonical, and the other represents the linebreak in the current source document from which the editor is working.

**Comment by joeytakeda** [comment]
Yes this makes sense. But as far as I can tell, these new line breaks are just being automatically produced.

**Comment by martindholmes** [comment]
Isn't that because the original code used actual linebreaks rather than tags?

**Comment by telic** [comment]
Yes, the non-typed `<lb/>` are the lineation of IML as edited. These are the breaks that we will want to use for display and they may not consistently correspond to TLNs or QLNs even in the OS texts.

**Comment by telic** [comment]
The confusion would be lessened by including `break="no"` on the TLNs, but @JanelleJenstad specifically asked us not to do this. IIRC, she had concerns about readability of the TEI. It seems to me the readability problems could quite easily be solved by putting the `lb`s on their own line though, eg from MV_M:
```
<l>
  <lb type="tln" n="2" break="no"/>
  <lb type="qln" n="3" break="no"/>
  In sooth I know not why I am so sad,
</l>
<l>
  <lb type="tln" n="3" break="no"/>
  <lb type="qln" n="4" break="no"/>
  It wearies me, you say it wearies you;
</l>
```

**Comment by joeytakeda** [comment]
This makes sense to me, other than the `@break`. As far as I can tell, `@break` isn't an instruction to break or not -- definition from the TEI: "indicates whether or not the element bearing this attribute should be considered to mark the end of an orthographic token in the same way as whitespace."

**Comment by joeytakeda** [comment]
To be clear: this would only apply in the case of verse lines, correct? Is the editor's decision to add a line break in one place or another in prose significant?

**Comment by martindholmes** [comment]
`@break` is intended to assist tokenizers processing the text; it tells you whether a hyphen is intra- or inter-word.

**Comment by telic** [comment]
Untyped `<lb/>` are not relevant in *modern* prose lines (and the conversion removes them). In OS texts however, the editors lineation is always significant, in both prose and verse (and the default "uncertain" mode from the OC example).

**Comment by telic** [comment]
Yes, my confusion on the use of `@break`; that's a separate issue.

**Comment by joeytakeda** [comment]
Ah, I see. So, the `<lb/>`s in prose in OS texts reflect the lineation of the copy-text, which was originally recorded by actual linebreaks. That makes sense to me.
